rest/v10/index.ts

```diff
@@ -380,6 +380,16 @@ export const Routes = {
 		return `/guilds/${guildId}/roles/${roleId}` as const;
 	},
 
+	/**
+	 * Route for:
+	 * - GET    `/guilds/{guild.id}/roles/member-counts`
```

Comment:
The spacing is only added to keep it uniform with the request methods (see below route for an example).

```suggestion
	 * - GET `/guilds/{guild.id}/roles/member-counts`
```